governance/polls/MIP9 Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022
+summary: Signal your support or opposition to prioritising onboarding HVBANK (Huntingdon Valley Bank Loan Syndication Collateral).
+discussion_link: https://forum.makerdao.com/t/mip6-huntingdon-valley-bank-loan-syndication-collateral-onboarding-application/14219
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Greenlight
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-04-18 16:00:00
+end_date: 2022-05-02 16:00:00
+---
+# Poll: Community Greenlight Poll - Investigate Implementation of HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - Monday, April 18 at 16:00 UTC
```

Comment:
This should say "Poll: Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022"